### ISSUE TITLE
Add filtering to directory view

### ISSUE THREAD

**Issue by jaredlockhart** [mozilla/experimenter#5958]
Let's add a horizontal filter bar on the top of the directory view (either above or below the status tabs, whichever looks better) with the following fields:

* Feature
* Application
* Owner
* Version

Each filter select should be a multiselect, and changing it should dynamically update the visible table.  Swapping status tabs should automatically apply the currently selected filters.  The displayed number of experiments in each tab should update as the filters change.  Currently applied filters should be reflected in the url as query parameters, and we’ll probably want to also put the currently selected status tab in there as well.

┆Issue is synchronized with this [Jiraserver Task](https://mozilla-hub.atlassian.net/browse/EXP-1494)

**Comment by data-sync-user** [comment]
➤ Les Orchard commented:

Whoops, accidentally changed the Reporter field.

**Comment by heres-maria** [comment]
I have verified that a horizontal filter bar on the top of the directory view (above the status tabs) has been added.

- The names of the fields are: Feature, Application, Owner, and Version.
- The fields are multiselect, more than one option can be added.
- Filters are applied dynamically.
- The status tab, filters and sorting columns are represented in the URL of the page.
- Swapping tabs keeps filters applied.
- The number of experiments in the status tab correctly changes as filters are applied.
Tested on Google Chrome 92.0.4515.131 and Firefox Release 91.0 (Build ID: 20210804193234). This feature is being tested as part of [PI-1153](https://mozilla-hub.atlassian.net/browse/PI-1153)/[EXP-1495](https://mozilla-hub.atlassian.net/browse/EXP-1495).
